src/pdf.scripting_api.js

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2020 Mozilla Foundation
```

Comment:
Given the much simplified packaging in the latest version, with this code now being bundled into the built `pdf.js` file, how about moving (and renaming) this file into the relevant folder instead?

E.g. something like `src/scripting_api/scripting.js` instead, with the paths updated accordingly in `src/pdf.js` and the gulpfile.